### PR TITLE
fix bugs from first draft for actionkit-rsvp #160

### PR DESCRIPTION
**Pull request by schuyler1d**

### CODE REVIEW

File: src/server/action_handlers/actionkit-rsvp.js [hunk 1]
```diff
@@ -73,13 +76,23 @@ export async function processAction(questionResponse, interactionStep, campaignC
...
+          if (err || (body && body.error)) {
+            console.error('error: actionkit event sign up failed', err, userData, body)
           } else {
+            if (httpResponse.headers && httpResponse.headers.location) {
+              const actionId = httpResponse.headers.location.match(/action_id=([^&]+)/)
+              if (actionId) {
+                //save the action id of the rsvp back to the contact record
+                customFields['processed_event_action'] = actionId[1]
+                await r.knex('campaign_contact')
+                  .where('campaign_contact_id', campaignContactId)
```
Comment (shakalee14) [inline src/server/action_handlers/actionkit-rsvp.js]: where does this `campaign_contact_id` come from? is this a new column, or should it be `campaign_contact.id`?

Comment (schuyler1d) [reply src/server/action_handlers/actionkit-rsvp.js]: good catch!

**Comment by mathemagica** [comment]
👍

**Review by shakalee14 — APPROVED** [review]

**Comment by shakalee14** [comment]
it would be nice to have a `HOWTO_INTEGRATE_WITH_ACTIONKIT.md`, thoughts @schuyler1d ? even if it was to just make sure certain variables are set?

**Pull request merged**